planning/development-strategy.md

```diff
@@ -25,86 +25,161 @@ A simple little website about trees.
 
-It should be clear what this project is for and what someone can expect from the
-web page.
+- [ ] _A Menu contians Home,Introduction,Content and Extras are displayed at the
+      top of the page._
 
-- [ ] _there is a section directly under the navbar describing the project's
-      goals_
+### Menu HTML
+
+-[ ] Navbar:Home,Introduction,Content,Extras(issue)
+
+### Menu CSS
+
+- [ ] style elements, hover (issue)
+- [ ] place elements using flex (issue)
+
+---
+
+## Introduction (milestone)
+
+> "As a user I can read an introduction to this project" It should be clear what
+> this project is for and what someone can expect from the web page.
+
+- [ ] _there is a section "Introduction" in the navbar describing the project's
+      goal_
 
 ### Introduction: HTML
 
-- a section with some general information about the project and helpful links
+- a section with some general information about the project and helpful
+  links(issue)
 
 ### Introduction: CSS
 
-- styling to make the text look fancy
+- styling to make the text look fancy.(issue)
 
 ---
 
-## Main Info
+## Content (milestone)
 
 > "As a user I can learn about trees"
 
-There should be enough information that the user can learn the basics about
-trees.
+Since it is a home page we don't place much content but there should be links to
+the content with clear description.
+
+- [ ] _A section with some general information about the uses of trees,tree
+      health and social impact_(issue)
+- [ ] _there are links to different sources providing content about trees_
+
+### Content: HTML
+
+- a div with title and paragraph explaining why Batman is the best
+  superhero(issue)
+
+### Content: CSS
+
+- [ ] hover effect on links (issue)
+- [ ] styling to make the content look nice(issue)
+
+## Extra Info (milestone)
+
+> "As a user I can learn even more about trees" Some more links for students who
+> want to go beyond the basics.
+
+- [ ] _there is an aside with links for further reading_
+
+### Extra Info: HTML
+
+- An aside with links for digging deeper, some helpful text and a reference link
+  (issue)
+
+### Extra Info: CSS
+
+- styling to position the aside (issue)
+- styling to style the text (issue)
+
+## Main Info (milestone)
+
+> "As a user I can learn about trees" There should be enough information that
+> the user can learn the basics about trees.
 
 - [ ] _there is a main text in the page with helpful info and links_
+- [ ] _there are two images in the left side of the main page_
 
 ### Main Info: HTML
 
-- Main text: a section with some general information about the project
-- Extra Info Links: a list of links
+- Main text: a section with some general information about the project.(issue)
+
+- Extra Info Links: a list of links (issue)
 
 ### Main Info: CSS
 
-- style the text
-- styling to format the list
+- style the text (issue)
+
+## Images (milestone)
+
+> "As a user I want to see the images of Tree's Health/social impact"
```

Comment:
Please put the User story as they are in backlog

Reply:
we have pictures in the design so i added here.